rust/sdk/Cargo.toml

```diff
@@ -38,46 +41,57 @@ serde_cbor = "0.11.2"
 url = "2.5.0"
 tracing = "0.1.40"
 
-# This feature provides exponential backoff on requests
-tokio = { version = "1.34.0", optional = true }
+# Feature: with_tokio
+tokio = { version = "1.15.0", features = [
+    "rt-multi-thread",
+    "macros",
+    "fs",
+], optional = true }
+tokio-util = { version = "0.6.9", features = ["compat"], optional = true }
+rand = { version = "0.8.5", optional = true }
 
+[dev-dependencies]
+# This one is to make examples work
+asynchronous-codec = "0.7.0"
+structopt = "0.3.26"
```

Comment:
new issue for upgrading this to clap?

Reply:
I don't know how necessary it is since it's an example